src/components/sdk/VideoTileGrid/VideoTileGrid.stories.mdx

```diff
@@ -32,3 +32,7 @@ const App = () => (
 ## Props
 
 <Props of={VideoTileGrid} />
+
+```
```

Comment:
You can remove this, I know I think I mentioned to add documentation on this but the prop is self-exlpanatory here I think.